brainglobe_utils/image_io/save.py

```diff
@@ -51,9 +51,13 @@ def to_tiff(img_volume, dest_path):
 
     dest_path : str
         Where to save the tiff stack.
+
+    photometric: str
+        Color space of image (to pass to tifffile.imwrite)
+        Use 'minisblack' for grayscale and 'rgb' for rgb
```

Comment:
```suggestion
        Use 'minisblack' (default) for grayscale and 'rgb' for rgb
```
For extra clarity for users unfamiliar with Python function signatures?

Reply:
Sounds good! I added your suggestion